src/pca_gophish.yml

```diff
@@ -21,25 +21,29 @@
 
     - name: Create mount point for GoPhish persistent data volume
       file:
+        mode: 0644
         path: "{{ pca_gophish_composition_path }}/data"
         state: directory
 
     - name: Change GoPhish published port from 3380 to 80
       lineinfile:
+        mode: 0644
```

Comment:
I don't think that it is necessary to specify the mode here.  I thought this was required only for tasks that will generate a new file.  Do you remember @mcdonnnj?

Reply:
`lineinfile` is given a pass unless it has the `create` parameter.

Reply:
Ah you are right @jsf9k - I just reviewed my local pre-commit errors and there wasn't anything for that task- I just got overzealous when I was adding those modes.  Removed in https://github.com/cisagov/pca-gophish-composition-packer/pull/22/commits/534bdd2914cc8a4a2f27cc0df4d22e9dfce8b90e.